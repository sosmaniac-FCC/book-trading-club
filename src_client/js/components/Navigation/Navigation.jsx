import React from 'react';

import Tabulation from '../Tabulation/Tabulation.jsx';

const Navigation = () => (
    <nav className="nav-extended purple accent-3">
        <h4>
            <i className="fa fa-book"></i> Collect Books! <i className="fa fa-book"></i>
        </h4>
        <div className="nav-content purple accent-1">
            <Tabulation />
        </div>
    </nav>
);

export default Navigation;