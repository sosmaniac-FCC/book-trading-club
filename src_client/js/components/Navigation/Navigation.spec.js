// expect from jest_expect.js
/* global expect */

import React from 'react';
import { shallow, configure } from 'enzyme';
import enzymeAdapter from 'enzyme-adapter-react-16';
import toJSON from 'enzyme-to-json';

configure({ adapter: new enzymeAdapter() });

import Navigation from './Navigation.jsx';

test('Navigation section above the router content loads as expected.', () => {
    const component = shallow(<Navigation />);
    const tree = toJSON(component);
    expect(tree).toMatchSnapshot();
});

test('Navigation section above the router content should have no style attributes.', () => {
    const component = shallow(<Navigation />);
    expect(component.find('[style]')).toHaveLength(0);
});