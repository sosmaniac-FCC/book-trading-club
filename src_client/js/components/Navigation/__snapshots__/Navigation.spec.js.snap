// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation section above the router content loads as expected. 1`] = `
<nav
  className="nav-extended purple accent-3"
>
  <h3>
    <i
      className="fa fa-book"
    />
    <a>
      Knowledge!
    </a>
    <i
      className="fa fa-book"
    />
  </h3>
  <div
    className="nav-content purple accent-1"
  >
    <Connect(Tabulation) />
  </div>
</nav>
`;
